docs/collections/contribute.md

```diff
@@ -1,14 +1,116 @@
 
-# Contributing Collections
+# Contribute
+
+## Creating custom blocks
+
+### Build a block
```

Comment:
Suggest removing this heading since the difference between "create a block" and "build a block" is minimal.